### PR TITLE
App: get overrides from flatpak ref

### PR DESCRIPTION
**Pull request by danrabbit**

### CODE REVIEW

File: src/Permissions/Backend/App.vala [hunk 1]
```diff
@@ -36,13 +38,36 @@ public class Permissions.Backend.App : GLib.Object {
...
+                    var keys = key_file.get_keys (GROUP);
+
+                    foreach (var key in keys ) {
+                        var values = key_file.get_value (GROUP, key).split (";");
+                        foreach (var value in values) {
+                            if (value.length == 0) {
+                                break;
+                            }
+
+                            permissions.add ("%s=%s".printf (key, value));
```
Comment (danrabbit) [inline src/Permissions/Backend/App.vala:58]: Would it be dumb for this to just be
```suggestion
                        permissions.add (key_file.get_value (GROUP, key).replace (";", "="));
```

File: src/Permissions/Backend/App.vala [hunk 2]
```diff
@@ -172,4 +195,31 @@ public class Permissions.Backend.App : GLib.Object {
...
+
+        if (!key_file.has_group (GROUP)) {
+            return permissions;
+        }
+
+        try {
+            var keys = key_file.get_keys (GROUP);
+
+            foreach (var key in keys ) {
+                var values = key_file.get_value (GROUP, key).split (";");
```
Comment (tintou) [inline src/Permissions/Backend/App.vala:210]:
```suggestion
                var values = key_file.get_string_list (GROUP, key);
```

File: src/Permissions/Backend/App.vala [hunk 3]
```diff
@@ -172,4 +195,31 @@ public class Permissions.Backend.App : GLib.Object {
...
+        var permissions = new GenericArray<string> ();
+
+        if (!key_file.has_group (GROUP)) {
+            return permissions;
+        }
+
+        try {
+            var keys = key_file.get_keys (GROUP);
+
+            foreach (var key in keys ) {
```
Comment (tintou) [inline src/Permissions/Backend/App.vala:209]:
```suggestion
            foreach (unowned string key in keys) {
```

File: src/Permissions/Backend/App.vala [hunk 4]
```diff
@@ -172,4 +195,31 @@ public class Permissions.Backend.App : GLib.Object {
...
+                var values = key_file.get_value (GROUP, key).split (";");
+                foreach (var value in values) {
```
Comment (tintou) [inline src/Permissions/Backend/App.vala:211]:
```suggestion
                foreach (unowned value in values) {
```

**Pull request merged**